index.js

```diff
@@ -1,15 +1,15 @@
 // Copyright (c) Brock Allen & Dominick Baier. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
 
-import Log from './src/Log';
-import OidcClient from './src/OidcClient';
-import WebStorageStateStore from './src/WebStorageStateStore';
-import InMemoryWebStorage from './src/InMemoryWebStorage';
-import UserManager from './src/UserManager';
-import AccessTokenEvents from './src/AccessTokenEvents';
-import MetadataService from './src/MetadataService';
+export const Log = require('./src/Log');
+export const OidcClient = require('./src/OidcClient');
+export const WebStorageStateStore = require('./src/WebStorageStateStore');
+export const InMemoryWebStorage = require('./src/InMemoryWebStorage');
+export const UserManager = require('./src/UserManager');
+export const AccessTokenEvents = require('./src/AccessTokenEvents');
+export const MetadataService = require('./src/MetadataService');
 
-export {
+export default {
     Log,
     OidcClient,
     WebStorageStateStore,
```

Comment:
Why was it necessary to change this? If the package.json points to the compiled version, won't this file be ignored?